GUI/Forms/ExtractProgressForm.cs

```diff
@@ -60,15 +75,27 @@ protected override void OnShown(EventArgs e)
                         extractProgressBar.Style = ProgressBarStyle.Marquee;
                     }));
 
+                    Console.WriteLine($"Folder export started to \"{path}\"");
                     CalculateFilesToExtract(root);
-                    initialFileCount = filesToExtract.Count;
 
                     Invoke((Action)(() =>
                     {
                         extractProgressBar.Style = ProgressBarStyle.Continuous;
                     }));
 
-                    await ExtractFilesAsync().ConfigureAwait(false);
+                    if (Decompile)
+                    {
+                        foreach (var resourceType in ExtractOrder)
```

Comment:
Loop filesToExtractSorted directly?

Reply:
Thanks, I wouldn't have spotted this ever.

Reply:
Well crap, I forgot about that.